src/main/java/net/masterthought/cucumber/json/DocString.java

```diff
@@ -17,4 +17,8 @@
     public String getValue() {
         return value;
     }
+
+    public String getValueWithBreakingSpace() {
+        return value!=null?value.replaceAll("(\r\n|\r|\n|\n\r)", "<br/>"):value;
```

Comment:
don't manipulate data - you can format how they are not displayed but don't change